recipe/meta.yaml

```diff
@@ -11,11 +11,11 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   {{ hash_type }}: {{ hash_val }}
-  # patches:
-  #   - python35-compat.patch  # [py35]
+  patches:
+     - python35-compat.patch  # [py35]
```

Comment:
Ah, OK, thanks @pmlandwehr. If you wouldn't mind, I think it would be healthy to comment the this line with this information.